acdc/greaterthan/utils.py

```diff
@@ -212,6 +212,17 @@ def get_all_greaterthan_things(num_examples, metric_name, device="cuda"):
         test_mask=None,
         test_patch_data=test_patch_data)
 
+
+CIRCUIT = {
+    # "input": [None], # special case input
+    "0305": [(0, 3), (0, 5)],
+    "01": [(0, 1)],
+    "MEARLY": [(0, None), (1, None), (2, None), (3, None)],
+    "AMID": [(5, 5), (6, 1), (6, 9), (7, 10), (8, 11), (9, 1)],
```

Comment:
note: 6.9 wasn't there before

Reply:
Important catch - thanks

Reply:
Also it seems like Michael and Ollie do not patch the query inputs to the AMID heads at all!

**I think that we should add all connections from early MLPs to the queries here** - this isn't perfect but adding all Attention Heads from Layer 9 or earlier seems like a bad idea --- it will mean the True Positive Rate becomes really, really low